src/Model/Blog.php

```diff
@@ -374,7 +374,7 @@ public function getSettingsFields()
             '<a class="font-icon-info-circled toggle-description"></a>'
         );
 
-        $editorField = ListboxField::create('Editors', 'Editors', $members)
+        $editorField = ListboxField::create('Editors', _t(__CLASS__ . '.many_many_Editors', 'Editors') , $members)
```

Comment:
```suggestion
        $editorField = ListboxField::create('Editors', $this->fieldLabel('Editors')) , $members)
```
This should correctly pick up the localised strings. Please do this for the other fields as well.